.pre-commit-config.yaml

```diff
@@ -1,42 +1,21 @@
-# SPDX-FileCopyrightText: 2020 Diego Elio Pettenò
+# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries
```

Comment:
unless you consider this a complete rewrite, the original author attribution must remain.

Reply:
Also, since you are not under contract for this work, you can omit "for Adafruit Industries"

Reply:
Can totally update.

Would you actually keep these PRs or just recreate them? I made these so we could look and see what was the right path.

And haven't received a full explanation on when the `for Adafruit` should be there. In my opinion I'm doing it for Adafruit, whether in contract or not...